src/components/Utils/EmailFormat.js

```diff
@@ -0,0 +1,19 @@
+export const validateEmailFormat = (string) => {
+    
+    // initialize local variables
+    var result = true;
```

Comment:
use `let` instead of `var` https://stackoverflow.com/questions/762011/what-is-the-difference-between-let-and-var